noxfile.py

```diff
@@ -56,5 +60,7 @@ def lint(session):
 @nox.session(python=PYTHON_VERSIONS)
 @nox.parametrize("django", DJANGO_VERSIONS)
 def test(session, django: str):
-    session.run("pip", "install", f"django~={django}.0", "-e", ".[dev]")
+    session.run_always("pip", "install", "pdm")
+    session.run_always("pdm", "install", "--dev")
+    session.run_always("pip", "install", f"django~={django}.0")
```

Comment:
I'm not terribly happy about this, but it works for now

https://github.com/pdm-project/pdm/issues/259#issuecomment-1407595572